README.md

```diff
@@ -62,7 +62,22 @@ const options = {
 }
 
 ```
+### Using @fastify/env to configure other plugins
```

Comment:
I would move it to the `docs/Guides` folder instead

Reply:
This is the `@fastify/env` repo, not the core `fastify` repo.